sdcm/cluster.py

```diff
@@ -3708,14 +3700,14 @@ def wrapper(*args, **kwargs):  # pylint: disable=too-many-statements,too-many-lo
 
         node_list = kwargs.get('node_list', None) or cl_inst.nodes
         timeout = kwargs.get('timeout', None)
-        # remove all arguments which is not supported by BaseScyllaCluster.node_setup method
+        # remove all arguments which are not supported by BaseScyllaCluster.node_setup method
         setup_kwargs = {k: v for k, v in kwargs.items()
                         if k not in ["node_list", "check_node_health", "wait_for_db_logs"]}
 
-        _queue = queue.Queue()
+        startup_queue, startup_results = queue.Queue(), []
 
         @raise_event_on_failure
```

Comment:
hard to get failure if we catch all the exceptions

Reply:
The `raise_event_on_failure` creates SCT event on the caught exceptions.
We have handling mechanism on the SCT events.
So, what is the problem that should be solved here?